### ISSUE TITLE
<css>issue: "opacity:0;" not working in Koreader.

### ISSUE THREAD

**Issue by gwd0715** [koreader/koreader#11543]
* KOReader version:2024.03.1
* Device: Kindle Oasis 2

#### Issue
For a 
```html
<p style="opacity:0;">xxx</p> 
``` 
tag, its content is still rendered as "opacity:1;" on the screen. 

This is the Raw file:
<img width="1594" alt="image" src="https://github.com/koreader/koreader/assets/17922721/dcd04e7c-a3bc-4adc-ba16-9f99cb652ca3">

This is what it looks like on the ebook viewer:
<img width="587" alt="image" src="https://github.com/koreader/koreader/assets/17922721/235b2391-8517-4123-9f63-468161fc57e3">

This is the screenshot on Koreader:
![Reader_2024-03-13_070302](https://github.com/koreader/koreader/assets/17922721/57d04c80-9c41-4d5f-8218-d2ad674ce4f3)

**Comment by poire-z** [comment]
Yep, `opacity:` is not supported in crengine.
Like many other "fancy" CSS properties. The ones we support (and sometimes, with only a subset of the allowed values) at https://github.com/koreader/crengine/blob/master/crengine/src/lvstsheet.cpp#L148.

https://developer.mozilla.org/en-US/docs/Web/CSS/opacity
It feels not hard to parse, but it feels a bit hard to implement (it applies to background too, so there would be quite a bit of cascade and color tweaks...)
(We support oppacity/alpha as part of a color value specificiation though: https://github.com/koreader/crengine/pull/474.)
Feels not super essential to me - on our eInk devices, we mostly aim at legibility, so better full opaque than any intermediate.

Also feels not essential in your screenshot :)
If you really want to hide them, you can always help yourself with a style tweak:
`p[aria-hidden] { visibility: hidden; }`

**Comment by Frenzie** [comment]
For a hackier example, you could even match `[style*="opacity"]` to do similar things.

**Comment by gwd0715** [comment]
Thanks so much!
